### ISSUE TITLE
result is different

### ISSUE THREAD

**Issue by wynla** [gitgiter/ConvNCF-pytorch#1]
Why the difference between the result calculated by this code and the result in the paper is so big

**Comment by zcjia** [comment]
me too.

**Comment by arouna12** [comment]
yeah were you able to get the result on the paper, if so how please on pytorch?
